hrsync

```diff
@@ -14,6 +15,21 @@ Target="${2%/}"
 TargetHost=$3
 
 # ------------------------------------------------------------
+
+
+# if remote host is set, set SSHPort to user specified value else set it to 22
+if [ -n "$3" ]
+then
+	if [ -z "$4" ]
```

Comment:
Well @martinhanes, I agree with the objections you raised and with your proposal. I'll just merge code that implements your solution for non standard ssh ports.

Reply:
i had similair thoughts at the start:
(for example the standard rsync syntax for ssh servers is the other way around (which confused me at first) ( root@example.com:/root/Documents))
(this might also be a point why **not** to use the proposed
`./hrsync /home/user/Documents /root/Documents root@example.com:666`
in that way.

However i thought then we should also give proper option flags at least for the ssh option and maybe even input output.
Which would brake existing scripts.

So i thought i just go with the "spirit of the script" and add a 4th parameter.

With that said:
- i also wouldn't know where to start / how to ĉode it with either of the other proposals.
- i do not have any strong opinions or feelings (yet ^^) and personally i also could perfectly live with either of the other Soloutions also if they have an "unconsistent" syntax.

Reply:
Sure, I can see why you would have api which is similar with rsync's api. But if I had to choose betweeen port number as 4th parameter OR the ":" version, both not consistent with rsync, I would go with ":" version every time. That was a motivation behind my comment.

Consistent with rsync or not, hrsync should provide help / comment saying how to specify port number and it's up to the user to provide a port parameter (nor not) in the way hrsync accepts it. **Option flags are in my opinion the best solution** and I agree it would be very nice to be consistent with rsync, however I thought it's not worth the work yet, if we're not planning to add anything else there in foreseeable time :-) 

tl:dr; Providing port after ':' is used in many other tools, and it does not break existing api, and I would go with it.